### ISSUE TITLE
Train a model on thermal images

### ISSUE THREAD

**Issue by Ilbotre** [facebookresearch/detectron2#1233]
## ❓ How to train a model on thermal images for ADAS pourpose with detectron2?

note: actually I'm using this model "COCO-InstanceSegmentation/mask_rcnn_R_50_FPN_3x.yaml" trained on coco_2017 dataset which is a RGB dataset for obj detection. I'm really interested in fine-tune this model in order to do the same task but with thermal images that have just one channel, the thermal intensity rather than 3. Reading your docs it seems that I have just to change the config node, changing:
_C.MODEL.PIXEL_MEAN = [103.530, 116.280, 123.675]
_C.MODEL.PIXEL_STD = [1.0, 1.0, 1.0]
with the new mean and std w.r.t. my new thermal dataset, and
_C.INPUT.FORMAT = "BGR" 
to 
L (8-bit pixels, black and white)  ?
P (8-bit pixels, mapped to any other mode using a color palette)  ?
Is this sufficient in order to let understand by the net that the images are thermal and not RGB?

## ❓ How can I change the architecture of a model in detectron2?

note: I know how can I obtain a model in your framework but How can I adjust it for my tasks? such as modify a dimension of a hidden layer or just to remove some predicted class from the head of the model?

## ❓ According to your training metrics, what is the difference, for an obj detection model, between "cls_accuracy" and "fg_cls_accuracy"?

note: I plot all the metrics during training into TensorBoard but I don't understand the difference between them

**Comment by ppwwyyxx** [comment]
> Is this sufficient in order to let understand by the net that the images are thermal and not RGB?

you can write your own function to read image (https://detectron2.readthedocs.io/tutorials/data_loading.html#write-a-custom-dataloader) if it doesn't understand

> How can I change the architecture of a model in detectron2

there are some configs you can change and you can write your own https://detectron2.readthedocs.io/tutorials/write-models.html

> According to your training metrics, what is the difference, for an obj detection model, between "cls_accuracy" and "fg_cls_accuracy"

classification accuracy for foreground proposals
